_posts/francois_gerthoffert/2019-03-03-ZenCrepes-GitHub-Agile-Analytics-made-easy.md

```diff
@@ -0,0 +1,301 @@
+
+<figure>
+    <img src="{{site.urlimg}}francois_gerthoffert/zencrepes/report-and-search.png" />
+    <figcaption>Summary view in the issues tab</figcaption>
+</figure>
+
+Using faceted search users can build queries to filter down on elements being analyzed and displayed, providing views into the operation of multiple teams. 
+
+<figure>
+    <img src="{{site.urlimg}}francois_gerthoffert/zencrepes/report-and-search-list-view.png" />
+    <figcaption>List view in the Issues tab</figcaption>
+</figure>
+
+The list view contains the list of all issues resulting from the query. Clicking on the issue opens it in GitHub for further investigations/modifications.
+
+Two more tabs (Velocity & Burndown) are also available to view those metrics over the entire duration of the project.
+
+
+## Find the odd status
+
+Do we have any issues, part of closed milestones, which are still open? This shouldn’t be possible, right? Let’s check it out!
+
+<figure>
+    <img src="{{site.urlimg}}francois_gerthoffert/zencrepes/report-and-search-odd.png" />
+    <figcaption>Something to investigate</figcaption>
+</figure>
+
+So out of the ~3500 issues, 5 are in an incorrect state and worth further investigations. Those were last updated almost a year ago, likely forgotten by their team.
+
+
+## Assembling a team
+
+
+__Note of caution__: _The elements detailed below only works for teams, used at working together with a similar process, and should only be considered as one of the possible metrics (or as a starting point), but shouldn't be considered as an accurate metric._
+
+Let’s imagine you are assembling a scrum team for a major deliverable, how could you get a sense of their potential velocity at the start of the initiative, until the velocity stabilizes?
+
+Well, you can simply assemble their past velocity.
+
+<figure>
+    <img src="{{site.urlimg}}francois_gerthoffert/zencrepes/report-and-search-velocity.png" />
+    <figcaption>Rob, Dusan and Minh’s joint weekly velocity over the past year
+</figcaption>
+</figure>
+
+This is probably a very controversial statement to make. Those metrics are truly useful when used as part of a large amount of data, the more you narrow down your criteria, the less trustworthy it will be as it becomes more prone to variability (See [Statistical Regularity](https://en.wikipedia.org/wiki/Statistical_regularity)). 
+
+But it gives us a starting point, indications that can be used to get a sense of initial velocity for this newly assembled team. In a sense, not a precise metric, but better than no metric at all.
+
+For example, if I was to take the chart above, I would be tempted to use 10 points/week as a starting point. So for a first 2 weeks sprint, it is probable that the team will deliver up to 20 points, fairly unlikely the would deliver more than 30, quite likely they would deliver more than 15. Makes sense? Again not an accurate prediction, but some insights usable during sprint planning (better than no metrics at all).
```

Comment:
- "_So for a first **2-week** sprint..._" (singular)
- "_**Make** sense?_" (no 's')